cmd/chainbridge/deploy.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"encoding/hex"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/ChainSafe/ChainBridgeV2/keystore"
+	log "github.com/ChainSafe/log15"
+	"github.com/urfave/cli"
+)
+
+var deployCommand = cli.Command{
+	Action:   handleDeployCmd,
+	Name:     "deploy",
+	Usage:    "deploys test ganache",
+	Category: "TESTING",
+	Flags:    DeployFlags,
+	Description: "\tThe deploy command is used to deploy a ganache instance for testing\n" +
+		"\tTo add additional accounts, -testaccount\n" +
+		"\tTo only use custom accounts, -reset\n" +
+		"\tTo use a mnemonic, -mnemonic",
+}
```

Comment:
"deploy" should reference deployment of the contracts, not starting of ganache, to avoid confusion. Could you update this PR to use "ganache" or something?

Reply:
@GregTheGreek suggested the `deploy` name, i'm ok to change it to ganache, just would want consensus on this.

Reply:
I think this would be a subcommand of `deploy`, but we're not there yet.

Updates look good but please rename `deployCommand`

Reply:
@anthonychernyak

Reply:
I've renamed it to `ganacheCommand`